src/v1/models/tenant.ts

```diff
@@ -0,0 +1,43 @@
+import { DataTypes } from 'sequelize';
+import { AppDataSource } from '../config';
+
+export const Tenant = AppDataSource.define(
+  'tenant',
+  {
+    id: {
+      type: DataTypes.INTEGER,
+      primaryKey: true,
+      autoIncrement: true,
+    },
+    tenant_name: {
+      type: DataTypes.STRING,
+      allowNull: false,
+    },
+    tenant_type: {
+      type: DataTypes.STRING,
+      allowNull: false,
+    },
+    is_active: {
+      type: DataTypes.BOOLEAN,
+      allowNull: false,
+    },
+    status: {
+      type: DataTypes.ENUM('live', 'draft'),
+      allowNull: false,
+    },
+    created_by: {
+      type: DataTypes.STRING,
+      allowNull: false,
+    },
+    updated_by: {
```

Comment:
last_updated_by - column value

Reply:
sequalize will update the updat_at column if update query were happens. update_at

Reply:
this is for updated by and I wanted to update the column name